public/style.css

```diff
@@ -73,6 +73,11 @@ body {
   color: #112D4E;
 }
 
+.result-p{
```

Comment:
Unless having a scientific abbreviations like f = force, v = velocity, it's best practice to avoid abbreviations. So, would be better to just replace this with `result-paragraph`.